sdk/tables/azure-tables/src/serialization.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { EdmGuid } from "./EdmGuid";
+import { EdmInt64 } from "./EdmInt64";
+import { isUint8Array, encodeByteArray, decodeString } from "./utils/bufferSerializer";
+
+const Edm = {
+  DateTime: "Edm.DateTime",
+  Int64: "Edm.Int64",
+  Guid: "Edm.Guid",
+  Binary: "Edm.Binary"
+};
+
+export function serialize(obj: any): object {
```

Comment:
Could we write some tests for `serialize` and `deserialize` as part of this PR?  It should be easy enough to test these functions directly without having to worry about test recorders and the like and it would improve my confidence that this code is correct (and that if it gets broken later we have something to look at and debug that isn't an end to end live test).